bin/run.py

```diff
@@ -420,14 +439,16 @@ def process_run(run: Run):
                 if result.out:
                     data = crop_output(result.out)
 
-            judgemessage = run.feedbackdir / "judgemessage.txt"
-            judgeerror = run.feedbackdir / "judgeerror.txt"
             # Add data from feedbackdir.
             for f in run.feedbackdir.iterdir():
-                if f in [judgemessage, judgeerror]:
-                    continue
                 if f.name.startswith("."):
                     continue  # skip "hidden" files
+                if f.name in ["judgemessage.txt", "judgeerror.txt"]:
+                    continue
+                if f.name.startswith("judgeimage.") or f.name.startswith("teamimage."):
+                    data += f"{f.name}: {shorten_path(self.problem, f.parent) / f.name}\n"
+                    ensure_symlink(run.problem.path / f.name, f, output=True, relative=False)
+                    continue
```

Comment:
This only works for the output _validator_, right? What if I want to run the output _visualizer_ using `bt test`?

Reply:
since the visualizer can read the output of the validator it should probably only ever run after it... so no visualization for `bt test`

Reply:
Those two point don't seem mutually exclusive to me, though :stuck_out_tongue: Why can we not also run the output visualizer after running the output validator here?

Reply:
Oohh woops, I confused `bt test` with `bt run` :joy: I see now that `bt run <submission> <test_case>` works as I'd expect (as in, it copies the resulting image from `<feedback_dir>` to the problem's working directory), thanks :smile: